src/foundry/client/ui/dialog.d.ts

```diff
@@ -58,9 +63,9 @@ declare class Dialog<Options extends DialogOptions = DialogOptions> extends Appl
    */
   static override get defaultOptions(): DialogOptions;
```

Comment:
The new default value for `focus` should be documented in the docstring.

Reply:
Nice catch!